javascript/javascript1/week1/homework_soheib/houseyPricey.js

```diff
@@ -0,0 +1,40 @@
+let peterHouse = {
```

Comment:
can be declared as const. Really nice approach with the object

Reply:
Thank you :D